test/resources/EphemeralKeys.spec.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var stripe = require('../testUtils').getSpyableStripe();
+var expect = require('chai').expect;
+
+describe('EphemeralKey Resource', function() {
+  describe('create', function() {
+    it('Sends the correct request', function() {
+      stripe.ephemeralKeys.create(
+        {customer: 'cus_123'},
+        {stripe_version: '2017-05-25'}
+      );
+      expect(stripe.LAST_REQUEST).to.deep.equal({
+        method: 'POST',
+        url: '/v1/ephemeral_keys',
+        data: {
+          customer: 'cus_123',
+        },
+        headers: {
+          'Stripe-Version': '2017-05-25',
+        },
+      });
+    });
+
+    function versionTests() {
```

Comment:
Wow, this works!?

Seems a little unusual to wrap up `it` cases like this :) Do you think instead we could just have two helper functions for `testErrorsOnNoStripeVersion` and `sendsStripeVersion` and just invoke them from within a few duplicated `it` cases? It's a tiny bit more boilerplate, but seems less weird, and I think it'll play nicer with the test framework in case someone wants to run one individual test case.